$OpenBSD: patch-rights,v 1.2 2000/11/20 10:02:03 camield Exp $

This port wants to change the rights on PREFIX/ and PREFIX/bin to 02755.
Usually PREFIX is /usr/local. The rights on /usr/local and /usr/local/bin
should be 0755 according to /etc/mtree/4.4BSD.dist.

Also, use PREFIX/share/dnscache instead of /etc for dnsroots.global.

--- hier.c.orig	Mon Nov 20 10:37:25 2000
+++ hier.c	Mon Nov 20 10:40:02 2000
@@ -2,10 +2,12 @@
 
 void hier()
 {
-  c("/","etc","dnsroots.global",-1,-1,0644);
+  h(auto_home,-1,-1,0755);
+  d(auto_home,"bin",-1,-1,0755);
 
-  h(auto_home,-1,-1,02755);
-  d(auto_home,"bin",-1,-1,02755);
+  d(auto_home,"share",-1,-1,0755);
+  d(auto_home,"share/dnscache",-1,-1,0755);
+  c(auto_home,"share/dnscache","dnsroots.global",-1,-1,0644);
 
   c(auto_home,"bin","dnscache-conf",-1,-1,0755);
   c(auto_home,"bin","tinydns-conf",-1,-1,0755);
